presto-native-execution/presto_cpp/main/thrift/presto_thrift.thrift

```diff
@@ -22,35 +22,276 @@ enum TaskState {
   ABORTED = 4,
   FAILED = 5,
 }
-
 enum ErrorType {
   USER_ERROR = 0,
   INTERNAL_ERROR = 1,
   INSUFFICIENT_RESOURCES = 2,
   EXTERNAL = 3,
 }
-
+enum ErrorCause {
```

Comment:
This file is auto generated based on idl, right?

Reply:
Discussed offline.

The IDL is generated based on Java classes. IDL will be generated automatically once we land changes to Drift. CMake generates thrift:: protocol classes during build process, they are not commited.

As a follow up we will also be generating the `ProtocolToThrift.h`